packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721.sol

```diff
@@ -78,8 +85,8 @@ contract OptimismMintableERC721 is ERC721Enumerable, IOptimismMintableERC721 {
     /**
      * @inheritdoc IOptimismMintableERC721
      */
-    function mint(address _to, uint256 _tokenId) external virtual onlyBridge {
-        _mint(_to, _tokenId);
+    function safeMint(address _to, uint256 _tokenId) external virtual onlyBridge {
+        _safeMint(_to, _tokenId);
```

Comment:
Not necessarily opposed, but I don't see anything about safeMint in the report?

Reply:
I thought it was in scope for L-08, where they mention the ERC721 bridges not using safeTransferFrom